src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -401,6 +402,19 @@ void PolicyManagerImpl::CheckPermissionsChanges(
                 CheckAppPolicy(this, pt_update, snapshot));
 }
 
+void PolicyManagerImpl::CheckPermissionsChangesAfterUpdate(
+    const policy_table::Table& update, const policy_table::Table& snapshot) {
+  const auto new_lock_screen_dismissal_enabled =
+      update.policy_table.module_config.lock_screen_dismissal_enabled;
+  const auto old_lock_screen_dismissal_enabled =
+      snapshot.policy_table.module_config.lock_screen_dismissal_enabled;
+  const bool lock_screen_dismisal_updated =
```

Comment:
seems overkill to use a variable here, should just be in the `if` statement.

Reply:
@jacobkeeler Thanks. Fixed in b07545f011f59e3f00851e5a6db7980a7cd76258